src/components/Seat/JetsSeat.test.js

```diff
@@ -0,0 +1,314 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { MockJetsContextProvider } from '../../__mocks__/MockJetsContext';
+
+import {
+  paramsData,
+  seatDataFirst,
+  seatDataBusiness,
+  seatDataPremium,
+  seatDataEconomy,
+  seatDataAisle,
+  seatDataEmpty,
+} from './__fixtures__';
+import { JetsSeat } from './index';
+
+const setup = ({ data = {}, config = {}, params = {}, events = {} } = {}) => ({
+  user: userEvent.setup(),
+  ...render(
+    <MockJetsContextProvider config={config} events={events} params={{ ...paramsData(params) }}>
+      <JetsSeat data={data} />
+    </MockJetsContextProvider>
+  ),
+});
+
+describe('JetsSeat', () => {
+  describe('when any seat type is rendered', () => {
+    it('should add the correct classes', async () => {
+      setup({ data: seatDataFirst() });
+
+      const wrapper = screen.getByText(/1A/).closest('.jets-seat');
+
+      // jets-seat twice due to component naming and seatType data
+      expect(wrapper).toHaveClass('jets-seat jets-seat jets-available');
```

Comment:
Apart from the type seat, which other types can we have?

Reply:
The other types are "aisle" and "empty" which are captured further down the file in the "when no seat is rendered" section.

There will be another commit to address the "index" type.